workspaces/tracer/package.json

```diff
@@ -25,6 +26,7 @@
   },
   "homepage": "https://github.com/NodeSecure/js-x-ray/tree/master/workspaces/tracer#readme",
   "dependencies": {
-    "@nodesecure/estree-ast-utils": "^3.0.0"
+    "@nodesecure/estree-ast-utils": "^3.0.0",
+    "meriyah": "^6.1.3"
```

Comment:
`meriyah` should be dev dep since its used only for typing ?

Reply:
You can't, how types work if that's devDep ? :p 

It's not a big deal since that's anyway a dep of JS-X-Ray